plugins/backend/rag-ai-backend/src/service/RagAiController.ts

```diff
@@ -85,6 +92,9 @@ export class RagAiController {
         message: 'No retrieval pipeline configured for this AI backend. ',
       });
     }
+    const { token } = await this.tokenManager.getToken();
```

Comment:
Few more of these left here and below. If you remove these, this should be good to go.

Actually the controller doesn't in that case need to token manager to be injected in at all so you could remove that also if you want.

Reply:
Ok it remember that i got errors on the deleteEmbeddings if no token submitted.
Giving a try...

Reply:
I have removed the Token Manager from the controller and the router as well.
Local testing is ok on my end.

Thank you for all theses advices @Xantier